scanner/updater/export.go

```diff
@@ -48,32 +48,34 @@ func Export(ctx context.Context, outputDir string) error {
 	defer func() {
 		closeErr := zstdWriter.Close()
 		if closeErr != nil {
-			zlog.Error(ctx).Err(closeErr).Msg("Failed to closing zstdWriter")
+			zlog.Error(ctx).Err(closeErr).Msg("Failed to close zstd writer")
 		}
 	}()
 
 	updaterSet, err := manual.UpdaterSet(ctx, nil)
 	if err != nil {
 		return err
 	}
-	outOfTree := [][]driver.Updater{
-		make([]driver.Updater, 0),
-	}
-	outOfTree = append(outOfTree, updaterSet.Updaters())
+	outOfTree := append(make([][]driver.Updater, 1), updaterSet.Updaters())
 
 	for i, uSet := range [][]string{
-		{"oracle", "photon", "suse", "aws", "rhcc"},
-		{"alpine", "rhel", "ubuntu", "osv", "debian"},
+		{"oracle", "aws", "rhcc"},
+		{"alpine", "rhel", "debian"},
+		{"ubuntu", "suse", "photon"},
+		{"osv"},
 	} {
 		jsonStore, err := jsonblob.New()
 		if err != nil {
 			return err
 		}
 
-		updateMgr, err := updates.NewManager(ctx, jsonStore, updates.NewLocalLockSource(), httpClient,
+		options := []updates.ManagerOption{
 			updates.WithEnabled(uSet),
-			updates.WithOutOfTree(outOfTree[i]),
-		)
+		}
+		if i < len(outOfTree) {
```

Comment:
Can you explain what's going on here? I don't understand this condition

Reply:
outOfTree only has one manual updater while **i** could be bigger that 0, here is just to avoid slice out of bound error